docs/source/start.rst

```diff
@@ -59,9 +59,24 @@ and Quil Compiler from the bare-bones package, you will have to install the prer
    You can also find the open source code for `quilc <http://github.com/rigetti/quilc>`__ and `qvm <http://github.com/rigetti/qvm>`__
    on GitHub, where you can find instructions for compiling, installing, and contributing to the compiler and QVM.
 
+Installing on Windows
+-------------------
```

Comment:
sphinx build complaining that the title underline here is too short. I think it needs to be extended at least as far as the text above it.

Reply:
🙄